mesonbuild/backend/vs2010backend.py

```diff
@@ -1782,6 +1817,9 @@ def path_normalize_add(path, lis):
                     ET.SubElement(inc_objs, 'Object', Include=s)
 
         ET.SubElement(root, 'Import', Project=r'$(VCTargetsPath)\Microsoft.Cpp.targets')
+        ext_tgt_grp = ET.SubElement(root, 'ImportGroup', Label='ExtensionTargets')
```

Comment:
I think that element could be created only `if masm`

Reply:
same as [above](https://github.com/mesonbuild/meson/pull/14171#discussion_r1940454689)